lab2/Pasichnichenko_Task2.py

```diff
@@ -0,0 +1,22 @@
+# Multi-paradigm programming languages Lab2 Illia Pasichnichenko IKM-221a
+print('Multi-paradigm programming languages Lab2')
+print('Illia Pasichnichenko IKM-221a')
+
+
+def get_float(message):
+    try:
+        user_input = float(input(message))
+        if isinstance(user_input, float):
```

Comment:
It's great that you tried checking by isinstance().
but from logic perspective it's meaningless. 
at the previous line you have casting to float.
So, there could be only 2 options: 
1. the program will fail with exception and the **if** will not be executed
2. you receive the value of float type. so, you don't need to check